README.md

```diff
@@ -71,35 +71,39 @@ This plugin does not work with non-app modules and/or library modules correctly.
 
 ## Version Compatibility
 
-| Android Gradle Plugin | Gradle DeployGate Plugin     |
-|:----------------------|:-----------------------------|
-| < 3.0.0               | 1.1.5 (No longer maintained) |
-| 3.3.x, 3.4.x, 3.5.x   | \>=2.0.0,\<2.6               |
-| 3.6.x                 | \>=2.1.0,\<2.6               |
-| 4.0.x                 | \>=2.2.0,\<2.6               |
-| 4.1.x                 | \>=2.3.0,\<2.6               |
-| 4.2.x                 | \>=2.4.0                     |
-| 7.0.x (Arctic Fox)    | \>=2.4.0                     |
-| 7.1.x (Bumblebee)     | \>=2.4.0                     |
-| 7.2.x (Chipmunk)      | \>=2.4.0                     |
-| 7.3.x (Dolphin)       | \>=2.4.0                     |
-| 7.4.x (Electric Eel)  | \>=2.4.0                     |
-| 8.0.x (Flamingo)      | \>=2.6.0                     |
-| 8.1.x (Giraffe)       | \>=2.7.0                     |
-| 8.2.x (Hedgehog)      | \>=2.7.0                     |
-| 8.3.x (Iguana)        | \>=2.8.0                     |
+| Android Gradle Plugin      | Gradle DeployGate Plugin     |
+|:---------------------------|:-----------------------------|
+| < 3.0.0                    | 1.1.5 (No longer maintained) |
+| 3.3.x, 3.4.x, 3.5.x        | \>=2.0.0,\<2.6               |
+| 3.6.x                      | \>=2.1.0,\<2.6               |
+| 4.0.x                      | \>=2.2.0,\<2.6               |
+| 4.1.x                      | \>=2.3.0,\<2.6               |
+| 4.2.x                      | \>=2.4.0                     |
+| 7.0.x (Arctic Fox)         | \>=2.4.0                     |
+| 7.1.x (Bumblebee)          | \>=2.4.0                     |
+| 7.2.x (Chipmunk)           | \>=2.4.0                     |
+| 7.3.x (Dolphin)            | \>=2.4.0                     |
+| 7.4.x (Electric Eel)       | \>=2.4.0                     |
+| 8.0.x (Flamingo)           | \>=2.6.0                     |
+| 8.1.x (Giraffe)            | \>=2.7.0                     |
+| 8.2.x (Hedgehog)           | \>=2.7.0                     |
+| 8.3.x (Iguana)             | \>=2.8.0                     |
+| 8.4.x (Jellyfish)          | \>=2.9.0                     |
+| 8.5.x (Koala)              | \>=2.9.0                     |
+| 8.6.x (Koala Feature Drop) | \>=2.9.0                     |
+| 8.7.x (Ladybug)            | \>=2.9.0                     |
```

Comment:
It's fine if we say these versions are supported since 2.9.0 though, did we add any fix to support the listed version? If no, these versions are implicitly supported since 2.8.0.

Reply:
> we add any fix to support the listed version?

No. We just added these versions to the acceptance test, and it passed without any fix.

> If no, these versions are implicitly supported since 2.8.0.

I see. I'll fix it.

Reply:
```suggestion
| 8.4.x (Jellyfish)          | \>=2.8.0                     |
| 8.5.x (Koala)              | \>=2.8.0                     |
| 8.6.x (Koala Feature Drop) | \>=2.8.0                     |
| 8.7.x (Ladybug)            | \>=2.8.0                     |
```